impeller/display_list/display_list_dispatcher.cc

```diff
@@ -827,6 +828,24 @@ static impeller::SamplerDescriptor ToSamplerDescriptor(
   return desc;
 }
 
+static impeller::SamplerDescriptor ToSamplerDescriptor(
```

Comment:
TBD: What is the difference between the flutter::DlFilterMode and flutter::DlImageSampling ?

Reply:
(Here I just convert the DlFilterMode into an impeller::SamplerDescriptor like we would for flutter::DlImageSampling

Reply:
ImageSampling represents more options than FilterMode. I haven't investigated why, but Skia only allows this subset for drawImageNine so we parallel what they allow in our DL API.